subiquity/server/tests/test_server.py

```diff
@@ -150,3 +151,26 @@ async def test_interactive_sections_one(self):
         mc = MetaController(make_app())
         mc.app.autoinstall_config['interactive-sections'] = ['network']
         self.assertEqual(['network'], await mc.interactive_sections_GET())
+
+
+class TestDefaultUser(SubiTestCase):
+    @patch('subiquity.server.server.ug_util.normalize_users_groups',
+           return_value=(None, None))
+    @patch('subiquity.server.server.ug_util.extract_default',
+           return_value=(None, None))
+    @patch('subiquity.server.server.user_key_fingerprints',
+           side_effect=Exception('should not be called'))
+    async def test_no_default_user(self, *patched_fns):
```

Comment:
Alternatively, you can specify the second argument of patch (a.k.a. `new`) if you don't want `patched_fns` - although it's a bit obscure.

```diff
    @patch('subiquity.server.server.ug_util.normalize_users_groups',
-           return_value=(None, None))
+           Mock(return_value=(None, None)))
    @patch('subiquity.server.server.ug_util.extract_default',
-           return_value=(None, None))
+           Mock(return_value=(None, None)))
    @patch('subiquity.server.server.user_key_fingerprints',
-           side_effect=Exception('should not be called'))
+           Mock(side_effect=Exception('should not be called')))
-   async def test_no_default_user(self, *patched_fns):
+   async def test_no_default_user(self):
```

> If [patch()](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) is used as a decorator and new is omitted, the created mock is passed in as an extra argument to the decorated function.

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch

Reply:
Nice!  Implemented.